app/bundles/CoreBundle/Assets/js/7.charts.js

```diff
@@ -225,125 +225,6 @@ Mautic.renderHorizontalBarChart = function(canvas) {
     Mautic.chartObjects.push(chart);
 };
 
-/**
- * Render vector maps
- *
- * @param mQuery element scope
- */
-Mautic.renderMaps = function(scope) {
```

Comment:
Replacing this public function looks like BC to me

Reply:
Indeed, I edited the PR and marked it as BC.

Reply:
That would mean we have to get this merged into Mautic 5.0.0-beta that should have been released last week. We are still waiting for a couple of PRs with BC breaks to be merged. But I don't want to extend that to other PRs otherwise we won't release it at all. How about to keep these methods and mark them as deprecated?